### PR TITLE
implementing mixinOrdering linter. addresses #90

### PR DESCRIPTION
**Pull request by shellscape**
I'll update docs when we're good with the functionality :)

### CODE REVIEW

**Comment by jwilsson** [comment]
I just restarted the AppVeyor build, it had just timed out.

File: lib/linters/mixin_ordering.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+// Array.findIndex polyfill
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex
+if (!Array.prototype.findIndex) {
```
Comment (jwilsson) [inline lib/linters/mixin_ordering.js]: We got a `lodash.findIndex` dependency included already

**Comment by jwilsson** [comment]
Great, as always. I think you forgot to add it to `linter.js` though :)

File: lib/linters/mixin_ordering.js (hunk 1 above)
Comment (shellscape) [inline lib/linters/mixin_ordering.js]: cool. ill kill the polyfill then

**Comment by shellscape** [comment]
Oye. I still need to get on this one.

**Comment by shellscape** [comment]
I need to completely refactor this one, and it's not a huge priority. closing for now. will open a new PR if/when I get around to the rebase and refactor.

**Pull request closed without merging**